src/test_servers.py

```diff
@@ -0,0 +1,26 @@
+"""Test client and echo server."""
+
+
+def test_one():
+    """Test if a string of less than one buffer echoes."""
+    from client import client
+    assert client(u'less') == u'less'
```

Comment:
Rather than having to prepend every string with `u` to coerce it to unicode in both Python 2 and Python 3, you can just do this at the top of your file:
`from __future__ import unicode_literals`

This coerces all string literals to unicode in both Python 3 and 2.